### PR TITLE
Translate Strict Mode

### PR DESCRIPTION
**Pull request by metinogurlu**
This PR translates the Advanced > Strict Mode page into Turkish.
Please review the document. Let me know about the possible improvements.

### CODE REVIEW

File: content/docs/strict-mode.md [hunk 1]
```diff
@@ -1,65 +1,65 @@
...
+`findDOMNode` ayrıca sınıf bileşenlerinde de kullanılabilir ancak bu, bir ebeveynin belirli alt öğelerin render edilmesine izin vererek soyutlama düzeylerini kırıyordu. Bir üst öğe DOM düğümüne erişebileceği için bir bileşenin uygulama ayrıntılarını değiştiremeyeceğiniz bir kodun yeniden düzenlenmesi (refactoring) tehlikesi oluşturur. `findDOMNode` sadece ilk alt öğeyi döndürür fakat Fragment'ler kullanılarak bir bileşenin birden fazla alt öğe render etmesi mümkündür. `findDOMNode` tek seferlik okuma API'sidir. Sadece istendiğinde cevap verir. Bir alt bileşen farklı bir farklı bir düğüm render ediyorsa, bu değişikliği ele almanın bir yolu yoktur. Bu nedenle `findDOMNode` sadece bileşenler asla değişmeyen tek bir DOM düğümü döndürürse işe yarar.
```
Comment (alioguzhan) [inline content/docs/strict-mode.md:58]: `ebeveyn` -> `üst eleman / öğe`
`kırıyordu` -> `bozuyordu`

File: content/docs/strict-mode.md [hunk 2]
```diff
@@ -73,53 +73,53 @@ class MyComponent extends React.Component {
...
+* **Commit** aşaması, React'ın değişiklikleri uyguladığı aşamadır. React DOM durumunda, React bu DOM düğümlerini ekler, günceller ve kaldırır. React ayrıca bu aşamada `componentDidMount` ve `componentDidUpdate` gibi yaşam döngülerini çağırır.
 
-The commit phase is usually very fast, but rendering can be slow. For this reason, the upcoming concurrent mode (which is not enabled by default yet) breaks the rendering work into pieces, pausing and resuming the work to avoid blocking the browser. This means that React may invoke render phase lifecycles more than once before committing, or it may invoke them without committing at all (because of an error or a higher priority interruption).
+Commit aşaması genellikle hızlıdır fakat render aşaması yavaş olabilir. Bu nedenle, yakında gelecek eşzamanlı (concurrent) yöntem (henüz varsayılan olarak etkin değil) render etme işini parçalara ayırır, tarayıcıyı engellememek için işi duraklatır ve devam ettirir. Bu, React'ın render aşaması yaşam döngülerini commit aşamasına hiç geçmeden (bir hata veya daha yüksek öncelikli kesinti nedeniyle) çağırabileceği ya da commit aşamasından önce bir kereden fazla çağırabileceği anlamına gelir.
```
Comment (alioguzhan) [inline content/docs/strict-mode.md:86]: `yontem` -> `modu`

File: content/docs/strict-mode.md [hunk 3]
```diff
@@ -1,65 +1,65 @@
 ---
 id: strict-mode
-title: Strict Mode
+title: Strict Yöntemi
```
[the remark this answers is not in the document]
Comment (metinogurlu) [reply content/docs/strict-mode.md]: It definitely makes more sense. I just saw the use of the "Strict Yöntemi" in [this commit](https://github.com/reactjs/tr.reactjs.org/commit/24611dc64a80315c02a2c6a3c8c91b0b948539ba) and prefered this for compatibility. Okay, I changed the "strict yöntemi" to "strict modu" now and also updated the PR with the change in the nav.yml file too.

**Comment by metinogurlu** [comment]
@alioguzhan requested changes is ok.

File: content/docs/strict-mode.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (alioguzhan) [reply content/docs/strict-mode.md]: Ah, I missed that during the review. Can you also please update that file, too?

[the remark this answers is not in the document]
Comment (alioguzhan) [reply content/docs/strict-mode.md]: Nevermind, you already did that. Just saw that. Thanks! :)

**Pull request merged**